x-pack/plugins/uptime/public/components/common/charts/__tests__/duration_charts.test.tsx

```diff
@@ -7,7 +7,7 @@
 import React from 'react';
 import DateMath from '@elastic/datemath';
 import { DurationChartComponent } from '../duration_chart';
-import { MonitorDurationResult } from '../../../../../common/types';
+import { MonitorDurationResult } from '../../../../../common';
```

Comment:
AS much as i love short imports, but i think in this case, it make sense to keep types, runtimetype and constants separate. I fully realize this impact many files in this PR, so reverting it will be quite an effort. Let me know your thoughts about this? 

cc @andrewvc

Reply:
No worries - I had thought we discussed using just `common` in the past. Are you saying you'd want it reverted for just `types`, or `['types', 'runtime_types']`?

Reply:
ahan, i must have forgotten it, for me i think keeping types and constants make more sense, but if you guys feel different, i have no objection to importing them via 'common'.

Reply:
I've removed/reverted this in https://github.com/elastic/kibana/pull/55086/commits/d77978f3dfaaae4cd9f4853febc2f5ef96900df1, https://github.com/elastic/kibana/pull/55086/commits/7fff70f42b082b6048ba5fc238b3216d0156def0, and https://github.com/elastic/kibana/pull/55086/commits/04ce6cc3432223b4e4bfc04344b71333330b80ff.